client/utils/session.js

```diff
@@ -1,4 +1,11 @@
 const handleSession = (orgSlug, token, cookies) => {
+  /*
+  This function is created while implementation of remember me checkbox
+  It is used to check whether sessionStorage or cookies is used to store
+  the token. If session storage is used then it will clear the cookies
+  and returns the token with session - true else it returns the token
+  with session - false.
```

Comment:
what does the `-` stand for?

Why is the cookie removed? Does that happenwhen remember me checkbox is unchecked? Can you please clarify